<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Interface</class>
 <widget class="QMainWindow" name="Interface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>830</width>
    <height>458</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Interface</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#Interface{
background-image: url(:/sucai2/背景2.jpg);}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="font">
       <font>
        <family>仿宋</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <italic>false</italic>
        <bold>true</bold>
        <underline>false</underline>
       </font>
      </property>
      <property name="title">
       <string>餐桌情况</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="2" column="4" rowspan="5">
        <widget class="QLabel" name="image">
         <property name="styleSheet">
          <string notr="true">image: url(:/sucai2/前台素材.png);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="0" column="4" rowspan="2">
        <widget class="QLabel" name="label">
         <property name="styleSheet">
          <string notr="true">image: url(:/new/prefix1/image/logo1.png);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="6" column="0" colspan="3">
        <widget class="QLabel" name="timelabel">
         <property name="font">
          <font>
           <family>仿宋</family>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>         0000-00-00 00:00:00 星期日</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QPushButton" name="table7">
         <property name="font">
          <font>
           <family>李旭科书法</family>
           <pointsize>36</pointsize>
          </font>
         </property>
         <property name="text">
          <string>桌7</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QPushButton" name="table8">
         <property name="font">
          <font>
           <family>李旭科书法</family>
           <pointsize>36</pointsize>
          </font>
         </property>
         <property name="text">
          <string>桌8</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QPushButton" name="table9">
         <property name="font">
          <font>
           <family>李旭科书法</family>
           <pointsize>36</pointsize>
          </font>
         </property>
         <property name="text">
          <string>桌9</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="table4">
         <property name="font">
          <font>
           <family>李旭科书法</family>
           <pointsize>36</pointsize>
          </font>
         </property>
         <property name="text">
          <string>桌4</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="table5">
         <property name="font">
          <font>
           <family>李旭科书法</family>
           <pointsize>36</pointsize>
          </font>
         </property>
         <property name="text">
          <string>桌5</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QPushButton" name="table6">
         <property name="font">
          <font>
           <family>李旭科书法</family>
           <pointsize>36</pointsize>
          </font>
         </property>
         <property name="text">
          <string>桌6</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="table1">
         <property name="font">
          <font>
           <family>李旭科书法</family>
           <pointsize>36</pointsize>
          </font>
         </property>
         <property name="text">
          <string>桌1</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="table2">
         <property name="font">
          <font>
           <family>李旭科书法</family>
           <pointsize>36</pointsize>
          </font>
         </property>
         <property name="text">
          <string>桌2</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="table3">
         <property name="font">
          <font>
           <family>李旭科书法</family>
           <pointsize>36</pointsize>
          </font>
         </property>
         <property name="text">
          <string>桌3</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>830</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>其他功能</string>
    </property>
    <addaction name="action1"/>
    <addaction name="action2"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action1">
   <property name="text">
    <string>添加菜单</string>
   </property>
  </action>
  <action name="action2">
   <property name="text">
    <string>修改菜单</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
